linera-storage/src/db_storage.rs

```diff
@@ -895,16 +913,17 @@ where
         block_exporter_id: u32,
     ) -> Result<Self::BlockExporterContext, ViewError> {
         let root_key = bcs::to_bytes(&BaseKey::BlockExporterState(block_exporter_id))?;
-        let store = self.store.open_exclusive(&root_key)?;
+        let store = self.database.open_exclusive(&root_key)?;
         Ok(ViewContext::create_root_context(store, block_exporter_id).await?)
     }
 }
 
-impl<Store, C> DbStorage<Store, C>
+impl<Database, C> DbStorage<Database, C>
 where
-    Store: KeyValueStore + Clone + Send + Sync + 'static,
+    Database: KeyValueDatabase + Clone + Send + Sync + 'static,
```

Comment:
Is there a single instance where a `KeyValueDatabase` does not have a corresponding `::Store: KeyValueStore`?